app/controllers/responses_controller.rb

```diff
@@ -20,7 +20,8 @@ def show
     end
 
     def index
-        @responses = Response.all
+        @q = @form.responses.ransack(params[:q])
+        @responses = @q.result.reverse_order
```

Comment:
I would suggest using `order` instead of `reverse_order` https://apidock.com/rails/ActiveRecord/QueryMethods/order

Using `reverse_order` just flips the array where `order` will allow you to specify what you want to order the elements by.

Reply:
See my next comment